spec/configuration.yml.example

```diff
@@ -7,11 +7,11 @@ root:
 user:
   host: localhost
   username: LOCALUSERNAME
-  password:
+  password: test
```

Comment:
The default `travis` user has no password: http://docs.travis-ci.com/user/database-setup/#MySQL

I'm not sure how the tests pass with this change?